server/msteams/client.go

```diff
@@ -893,6 +909,91 @@ func (tc *ClientImpl) GetReply(teamID, channelID, messageID, replyID string) (*M
 	return convertToMessage(res, teamID, channelID, ""), nil
 }
 
+func GetMessageFromJSON(data []byte, teamID, channelID, chatID string) (*Message, error) {
```

Comment:
What are your thoughts on this comment? @jespino

Reply:
I don't love `util.go` files or `utils` packages, because that means nothing, and normally they end up being a bucket full of entropy, so I prefer to have another filename that give meaning to the content, even if that means the file or package only contains that function.